source/CCustomOpcodeSystem.cpp

```diff
@@ -1520,6 +1519,22 @@ namespace CLEO
 		return OR_CONTINUE;
 	}
 
+	//2000=1, %1d% = get_cleo_arg_count
+	OpcodeResult __stdcall opcode_2000(CRunningScript* thread)
+	{
+		auto cs = reinterpret_cast<CCustomScript*>(thread);
+
+		ScmFunction* scmFunc = ScmFunction::Get(cs->GetScmFunction());
+		if (scmFunc == nullptr)
+		{
+			SHOW_ERROR("Quering argument count without preceding CLEO function call in script %s\nScript suspended.", cs->GetInfoStr().c_str());
+			return thread->Suspend();
+		}

```

Comment:
At some point this opcode might be extended to return number of arguments passed to current script with 004f, 0913, 0a92, etc.

But good for now